clients/admin-ui/src/types/api/models/Property.ts

```diff
@@ -2,11 +2,13 @@
 /* tslint:disable */
 /* eslint-disable */
 
-import { PropertyType } from "./PropertyType";
+import type { PropertyType } from "./PropertyType";
 
+/**
+ * A base template for all other Fides Schemas to inherit from.
+ */
 export type Property = {
-  key: string;
   name: string;
-  type: string;
-  experiences?: Array<string>;
+  type: PropertyType;
+  key?: string;
```

Comment:
should this really be optional?

Reply:
The key is auto-generated on the backend on creation so it's optional to be able to create with just a name and type-- every property has a key after it's created.